### ISSUE TITLE
Reset after second message of MQTT

### ISSUE THREAD

**Issue by MikeGitHubRPi** [widapro/wledPixel#57]
Hello.
I'm using MQTT to send a news from Node Red to wledPixel but after the second message sent to the panel it resets.
It doesn't matter how log I wait to send the second or the third message.

Any idea?
Thank you

**Comment by martusi61** [comment]
Yes, I also had the same problems and the developer solved them by increasing the maximum length of the message to 100 characters, I think, so, check your length.

**Comment by MikeGitHubRPi** [comment]
Thank you very much martusi61. Now it works!!!

**Comment by martusi61** [comment]
Happy to help you!
